src/babylon.scene.ts

```diff
@@ -3,6 +3,72 @@
         dispose(): void;
     }
 
+    export class SceneAssetContainer {
+            public scene: Scene;
+
+            // Config
+            public useDelayedTextureLoading:boolean;
+            public autoClear:boolean;
+            public clearColor:Color4;
+            public ambientColor:Color3;
+            public metadata:any;
+            public activeCameraID:string;
+
+            // Physics
+            public gravity:Vector3;
+            public physicsGravity:Nullable<Vector3>;
+            public physicsPluginName:string;
+            public enablePhysics: boolean;
+            public collisionsEnabled:boolean;
+            public workerCollisions:boolean;
+
+            // Fog
+            public fogMode:number;
+            public fogColor:Color3;
+            public fogStart:number;
+            public fogEnd:number;
+            public fogDensity:number;
+
+            // Animation
+            public autoAnimate:boolean;
+            public autoAnimateFrom:number;
+            public autoAnimateTo:number;
+            public autoAnimateLoop:number;
+            public autoAnimateSpeed:number;
+
+            // Skybox
+            public createDefaultSkybox:boolean;
+            public environmentTexture:BaseTexture;
+            public skyboxBlurLevel:number;
+
+            // Objects
+            public cameras = new Array<Camera>();

```

Comment:
These are all the objects I will make sure a remove* and add* method exists for within the scene if they don't already exist.